### PR TITLE
Enable Cancer SV observations view

### PR DESCRIPTION
**Pull request by ivadym**
This PR adds a functionality or fixes a bug:
- Cancer observations (both somatic and germline) are not appearing in the SV view in Scout (https://github.com/Clinical-Genomics/scout/issues/4341).

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
- Verify parsing of local observations from VCF

**Expected outcome**:
- Values correspond to the ones in the SVDB VCF:
Screenshot 2024-01-16 at 09 21 21

**Review:**
- [x] code approved by CR
- [x] tests executed by CR, VI

### CODE REVIEW

**Comment by ivadym** [comment]
No changes to the view after deploying on Hasta... 🤔

**Comment by northwestwitch** [comment]
> No changes to the view after deploying on Hasta... 🤔 It may be something less obvious hiding the cancer observations

That's because we have two instances of Scout:
1. The hasta one if for running command line stuff, for instance load cases, export panels etc
2. The cg-vm1 is the web portal

So to test this one one should deploy on cg-vm1. I've just done it and it looks good:

Main branch:

image


This branch: 🆗 

image

File: scout/server/blueprints/variant/templates/variant/variant_details.html [hunk 1]
```diff
@@ -216,7 +216,7 @@
             <td>{{ variant.local_obs_hom_old|default('N/A') }}</td>
             <td>{% if variant.local_obs_old_freq %} {{variant.local_obs_old_freq|round(6)}} {% elif variant.local_obs_old_nr_cases and variant.local_obs_old %} {{ (variant.local_obs_old/variant.local_obs_old_nr_cases)|round(5) }} {% elif variant.local_obs_old_nr_cases %} 0 / {{variant.local_obs_old_nr_cases}} {% else %} N/A {% endif %}</td>
           </tr>
-          {% if variant.category == "cancer" %}
+          {% if variant.category in ['cancer', 'cancer_sv'] %}
```
Comment (northwestwitch) [inline scout/server/blueprints/variant/templates/variant/variant_details.html:219]: 💯

**Review by northwestwitch — APPROVED** [review]
Great fix!!! 

Don't forget to update the changelog file before merging, so all tests will pass

**Comment by ivadym** [comment]
> Great fix!!!
> 
> Don't forget to update the changelog file before merging, so all tests will pass

Thanks for the help @northwestwitch! ⭐

**Comment by northwestwitch** [comment]
> Thanks for the help @northwestwitch! ⭐

Thank you. I think this PR is ready to be merged now

**Pull request merged**